app/locale/fr_FR/Oyst_OneClick.csv

```diff
@@ -146,3 +146,6 @@
 "Actions","Actions"
 "Delete","Effacer"
 "Download","Télécharger"
+
+"Impossible to issue a refund transaction because the capture transaction does not exist","Impossible de lancer le remboursement de la commande car celle ci n'a pas été capturée"
+"There is already one credit memo in the queue.","Il existe déjà une note de crédit dans la file d'attente."
```

Comment:
Could you change this :
Il existe déjà une note de crédit dans la file d'attente.
with
Il y a déjà un remboursement en cours.